storage-node/packages/cli/src/commands/dev.ts

```diff
@@ -142,4 +142,101 @@ const init = async (api: RuntimeApi): Promise<any> => {
   return check(api)
 }
 
-export { init, check, aliceKeyPair, roleKeyPair, developmentPort }
+// Using sudo create initial storage lead and worker with given keys taken from env variables.
+// Used to quickly setup a storage provider on a new chain before a council is ready.
+const makeMemberInitialLeadAndStorageProvider = async (api: RuntimeApi): Promise<any> => {
+  const sudoKey = getKeyFromAddressOrSuri(api, process.env.SUDO_URI)
+  const memberId = parseInt(process.env.MEMBER_ID)
+  const memberController = getKeyFromAddressOrSuri(api, process.env.MEMBER_URI).address
+  const leadAccount = memberController
+  const workerAccount = process.env.WORKER_ACCOUNT
```

Comment:
At the beginning I tried using `WORKER_URI` as provided in PR description and this resulted in some arbitrary(?) role key (`5C4hrf...yUpnhM`).

When using `WORKER_ACCOUNT=5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty` (Bob) the result is as expected.

Reply:
Yes I think it derived account from an empty string maybe. The description in the PR was incorrect. :)
I'll make some general improvements on input validation.